content/accounts.md

```diff
@@ -0,0 +1,702 @@
+  }
+
+  const { first_name, last_name } = user.services.facebook;
+  user.initials = first_name[0].toUpperCase() + last_name[0].toUpperCase();
+
+  // Don't forget to return the new user object at the end!
+  return user;
+});
+```
+
+Note that the `user` object provided doesn't have an `_id` field yet. If you need to do something with the new user's ID inside this function, a useful trick can be to generate the ID yourself:
+
+```js
+// Generate a todo list for each new user
+Accounts.onCreateUser((options, user) => {
+  // Generate a user ID ourselves
+  user._id = Random.id(); // Need to add the `random` package
+
+  // Use the user ID we generated
+  Lists.createListForUser(user._id);
+
+  // Don't forget to return the new user object at the end!
+  return user;
+});
+```
+
+<h3 id="dont-use-profile">Don't use profile</h3>
+
+There's a tempting existing field called `profile` that is added by default when a new user registers. This field was historically intended to be used as a scratch pad for user-specific data - maybe their image avatar, name, intro text, etc. Because of this, **the `profile` field on every user is automatically writeable by that user from the client**. It's also automatically published to the client for that particular user.
+
+It turns out that having a field writeable by default without making that super obvious might not be the best idea. There are many stories of new Meteor developers storing fields such as `isAdmin` on `profile`... and then a malicious user can easily set that to true whenever they want, making themselves an admin.
+
+Rather than dealing with the specifics of this field, it can be helpful to just ignore its existence entirely. You can safely do that as long as you deny all writes from the client:
+
+```js
+// Deny all client-side updates to user documents
+Meteor.users.deny({
+  update() { return true; }
+});
+```
+
+Even ignoring the security implications of `profile` in particular, it isn't a good idea to put all of your app's custom data onto one field. As discussed in the [Collections article](collections.html#schema-design), Meteor's data transfer protocol doesn't do deeply nested diffing of fields, so it's a good idea to flatten out your objects into many top-level fields on the document.
+
+<h3 id="publish-custom-data">Publishing custom data</h3>
+
+If you want to access the custom data you've added to the `Meteor.users` collection in your UI, you'll need to publish it to the client. Mostly, you can just follow the advice in the [Data Loading](data-loading.html#publications) and [Security](security.html#publications) articles.
+
+The most important thing to keep in mind that user documents are certain to contain secret data about users. In particular, the user document includes hashed password data and access keys for external APIs. This means it's critically important to [filter the fields](http://guide.meteor.com/security.html#fields) of the user document that you send to any client.
```

Comment:
typo?
The most important thing to keep in mind that user documents -> The most important thing to keep in mind is that user documents